client/shared/src/components/LastSyncedIcon.tsx

```diff
@@ -2,13 +2,21 @@ import classNames from 'classnames'
 import format from 'date-fns/format'
 import CloudSyncOutlineIcon from 'mdi-react/CloudSyncOutlineIcon'
 import React from 'react'
+
 import styles from './LastSyncedIcon.module.scss'
 
-export const LastSyncedIcon: React.FunctionComponent<{ lastSyncedTime: string }> = ({ lastSyncedTime }) => {
-    const formattedTime = format(Date.parse(lastSyncedTime), "yyyy-MM-dd'T'HH:mm:ss")
+export interface Props {
+    lastSyncedTime: string
+    className?: string
+}
+
+export const LastSyncedIcon: React.FunctionComponent<Props> = props => {
+    const formattedTime = format(Date.parse(props.lastSyncedTime), "yyyy-MM-dd'T'HH:mm:ss")
 
-    return (<CloudSyncOutlineIcon
-		className={classNames(styles.lastSyncedIcon, "icon-inline", "text-muted")}
-		data-tooltip={`Last synced: ${formattedTime}`}
-	/>)
+    return (
+        <CloudSyncOutlineIcon
+            className={classNames(props.className || styles.lastSyncedIcon, 'icon-inline', 'text-muted')}
```

Comment:
Why wouldn't we want both classes here?

```suggestion
            className={classNames(props.className, styles.lastSyncedIcon, 'icon-inline', 'text-muted')}
```

Reply:
Do'h. Was just testing this and noticed it was weird. Thank you 😄